numpy/tests/test_public_api.py

```diff
@@ -164,21 +165,21 @@ def test_NPY_NO_EXPORT():
     "compat",
     "compat.py3k",
     "conftest",
-    "core",
-    "core.arrayprint",
-    "core.defchararray",
-    "core.einsumfunc",
-    "core.fromnumeric",
-    "core.function_base",
-    "core.getlimits",
-    "core.memmap",
-    "core.multiarray",
-    "core.numeric",
-    "core.numerictypes",
-    "core.overrides",
-    "core.records",
-    "core.shape_base",
-    "core.umath",
+    "_core",
```

Comment:
All of these can be deleted instead of renamed I believe - they should simply not show up anymore.